e2e/multi-pipeline/s3template.yml

```diff
@@ -29,7 +29,7 @@ Resources:
     Type: AWS::S3::BucketPolicy
     Properties:
       PolicyDocument:
-        Version: 2012-10-17
+        Version: '2012-10-17'
```

Comment:
Why do we add `''` 🤔

Reply:
Without the quotes, the version is parsed as a Date instead of a String. This didn't seem quite right to me, since the policy language version is defined to be a string, even though CloudFormation appears to handle this properly.